### ISSUE TITLE
`dotnet new wasmbrowser` template papercuts

### ISSUE THREAD

**Issue by lambdageek** [dotnet/runtime#72624]
I installed `Microsoft.NET.Runtime.WebAssembly.Templates::7.0.0-preview.2.22152.2`, then did the following:

```console
cd /tmp
mkdir qa
dotnet new wasmbrowser
```

Papercuts:

1. The project file is called `browser.csproj`, not `<directory_name>.csproj` the way that other templates (e.g. `dotnet new console`) do it.
2. The `main.js` file contains 12 lines of actual code, and 300 lines of this:
   ```
      // SIG // Begin signature block
      // SIG // MIIopwYJKoZIhvcNAQcCoIIomDCCKJQCAQExDzANBglg
     ... (300 more lines of signature) ...
   ```

**Comment by lambdageek** [comment]
Looks like both of these are already fixed in `7.0.100-rc.1.22364.7`

**Comment by lewing** [comment]
I believe this is all fixed in main

```
lewing@MacBook-Pro backup % mkdir test-wasm
lewing@MacBook-Pro backup % cd test-wasm 
lewing@MacBook-Pro test-wasm % dotnet new wasmbrowser
The template "WebAssembly Browser App" was created successfully.

lewing@MacBook-Pro test-wasm % ls
Program.cs            app-support.js            index.html            main.js                runtimeconfig.template.json    test-wasm.csproj
```
no signatures in the js either.
